### PR TITLE
dxScheduler - don't show 'Done' button of appointment popup in read-only mode(T895434)

### PR DESCRIPTION
**Pull request by andrewmakarov**

### CODE REVIEW

File: js/ui/scheduler/appointmentPopup.js [hunk 1]
```diff
@@ -280,20 +280,24 @@ export default class AppointmentPopup {
         }
     }
 
-    _getPopupToolbarItems() {
+    _createPopupToolbarItems(doneButtonVisible) {
```
Comment (Yarovaya) [inline js/ui/scheduler/appointmentPopup.js]: isDoneButtonVisible?

File: playground/jquery.html [hunk 2]
```diff
@@ -1,11 +1,12 @@
 <!DOCTYPE html>
 <html>
+
 <head>
     <title>DevExtreme jQuery Example</title>
 
-    <link rel="stylesheet" type="text/css" href="../artifacts/css/dx.common.css"/>
+    <link rel="stylesheet" type="text/css" href="../artifacts/css/dx.common.css" />
```
Comment (Yarovaya) [inline playground/jquery.html]: Revert this changes

File: js/ui/scheduler/appointmentPopup.js (hunk 1 above)
Comment (andrewmakarov) [reply js/ui/scheduler/appointmentPopup.js]: I agree

File: testing/tests/DevExpress.ui.widgets.scheduler/appointmentPopup.tests.js [hunk 3]
```diff
@@ -113,6 +113,41 @@ const moduleConfig = {
...
+                dataSource: data,
+                views: ['week'],
+                currentView: 'week',
+                currentDate: new Date(2017, 4, 25),
+                editing: {
+                    allowUpdating: allowUpdatingValue
+                }
+            });
+
+            const assertText = `done button visible should be equal = ${allowUpdatingValue}`;
```
Comment (Yarovaya) [inline testing/tests/DevExpress.ui.widgets.scheduler/appointmentPopup.tests.js]: Done button visibility should be equal to ${allowUpdatingValue}

**Review by AntonSermyazhko — DISMISSED** [review]

**Review by AryamnovEugeniy — DISMISSED** [review]

**Review by AntonSermyazhko — APPROVED** [review]

**Review by Yarovaya — APPROVED** [review]

**Review by AryamnovEugeniy — APPROVED** [review]
